### ISSUE TITLE
Test failures in SIF collection

### ISSUE THREAD

**Issue by dpo** [JuliaSmoothOptimizers/CUTEst.jl#176]
I'm seeing test failures of the form
```
CUTEst selection tool: Test Failed at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:20
  Expression: np == length(CUTEst.select())
   Evaluated: 1305 == 1315
Stacktrace:
 [1] macro expansion at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:20 [inlined]
 [2] macro expansion at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
 [3] top-level scope at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:18
Variables: Test Failed at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:29
  Expression: np == n + length(selection)
   Evaluated: 1305 == 1315
Stacktrace:
 [1] macro expansion at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:29 [inlined]
 [2] macro expansion at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
 [3] macro expansion at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:22 [inlined]
 [4] macro expansion at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
 [5] top-level scope at /Users/dpo/dev/julia/JSO/CUTEst.jl/test/test_select.jl:18
```
They are probably due to an updated SIF collection?

**Comment by abelsiqueira** [comment]
Yes, the internal selection tool uses a different list of problems from
MASTSIF, so it returns an error. Maybe there is no need for such a check.

**Comment by dpo** [comment]
Fixed in #209.
